File: estus-zachary/areYouPlayingBanjo.js
// Check the first letter in the name for R or r
// return a string saying yes or no to playing the banjo

function areYouPlayingBanjo(name) {
  if(name[0] === 'R' || name[0] === 'r')
    return name + ' plays banjo';
  else
    return name + ' does not play banjo';
  return name;
}

File: mullins-roger/just-for-fun/are-you-playing-banjo.py
def are_you_playing_banjo(name):
    # Implement me!
    ret_string = ""
    if name[0] == "R" or name[0] == "r":
        ret_string = name + " plays banjo"
    else:
        ret_string = name + " does not play banjo"
    return ret_string
